index.js

```diff
@@ -26,6 +26,13 @@ commandDirectories.forEach(dir => {
   });
 });
 
+const scheduledJobsDirectory = './scheduledJobs';
+const scheduledJobFiles = fs.readdirSync(scheduledJobsDirectory).filter(file => file.endsWith('.js'));
+scheduledJobFiles.forEach((file) => {
+  const job = require(`${scheduledJobsDirectory}/${file}`);
+  job.start(client);
+});
+
```

Comment:
Maybe this could be placed inside the `client.once('ready')`? 🤔
https://github.com/BangtanAcademyCoderCrew/Project-BangPD/blob/d0af3a7e4bfe0c5e39ceff420a7740956553fe41/index.js#L36-L40